<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SYW</class>
 <widget class="QWidget" name="SYW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainAttribute">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="attribute1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="attribute2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="attribute3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Textlabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="attribute4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twentyBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>20级大狗粮</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4级小狗粮</string>
   </property>
  </widget>
  <widget class="QPushButton" name="againBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>再次刷取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewRatingBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查看评分</string>
   </property>
  </widget>
  <widget class="QLabel" name="marklabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backbtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
